<ui version="4.0" >
 <class>OptionsWidgetClass</class>
 <widget class="QWidget" name="OptionsWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptionsWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="grbCommonOptions" >
     <property name="title" >
      <string>Common notification options</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <item>
       <widget class="QCheckBox" name="chbEnableRosterIcons" >
        <property name="text" >
         <string>Roster</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnablePopupWindows" >
        <property name="text" >
         <string>Popup</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnableTrayIcons" >
        <property name="text" >
         <string>Tray</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chbEnableSounds" >
        <property name="text" >
         <string>Sound</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
